### ISSUE TITLE
FloorPlan creator for ESPresenseIPS

### ISSUE THREAD

**Issue by stan69b** [ESPresense/ad-espresense-ips#12]
Hey, the FloorPlan creator for ESPresenseIPS (https://github.com/stan69b/ESPresenseIPS-Floorplan-Creator) is now in a stable state.

This application allows one to : 
- Create a floorplan/Rooms easily
- Name each room
- Add your ESP32 devices inside each room
- Add z values for ESP32's, coverage radius and coverage circle color
- Show coverage of each ESP32 devices to see the exact coverage in each rooms
- Show coverage of ESP32 while you place them so you can find what are the best places.
- Export your floorplan to yaml format with room coordinates and esp32 devices coordinates (just copy past inside app.js)

A feedback would be most welcome, on the functional part mostly. 
For now if multiple esp32 in a room, they will all have the same name (room name) in the yaml, is this ok ? (i suppose not, but can't guess the right way).

Feel free to link this app in your readme if you wish.

(Funny fact, I have not tested the app myself, at least not for my home. I did however triple check every calculations. xD)

**Comment by GrimDeclaw** [comment]
Hi,
Great piece of software that really helps!
I think that there is a little indent issue in the roomplans : name and coordinates are not aligned
` - name: room1
        y1: 0
        x1: 3.6
        y2: 6.1
        x2: 8.3`

Functionnaly speaking it's great, I'll not talk about the pan as it is in your roadmap (my house is way longer than wide) but if I could add one thing it would be : While dragging a room, using the keyboard arrow to be more precise could be nice (I didn't look at the code but it should not be that hard to implement)

Once again, thanks for your great work!

**Comment by DTTerastar** [comment]
I just saw this!!! I've gotten swamped w/ bug reports on the firmware and WOW, this is super cool!

**Comment by DTTerastar** [comment]
Added to readme.  If you want to move it into this org i'm game.  I can't figure out how to get the mqtt functionality to connect.  I use no auth, so it might be intolerant of that.

**Comment by stan69b** [comment]
Yeah, we can move it to this org, it has been developed mainly for your plugin so it makes sense. I just don’t know what to do haha
For the MQTT part, you can open an issue if needed and I’ll look into it. (On the Floorplan creator repo rather than this one ;) )

**Comment by DTTerastar** [comment]
I think you need to give me admin for the repo to move it in.  I think the connection issue was handled separately.

**Comment by stan69b** [comment]
I tried transferring it to the organisation but got a Not found, before I screw things up, I added you as a cllaborator. Think you need to accept so I can set your rights.

**Comment by DTTerastar** [comment]
It moved it to: https://espresense.com/ESPresenseIPS-Floorplan-Creator/

**Comment by stan69b** [comment]
Is it all good ? Did I transfer to tue right place ? Haha

**Comment by DTTerastar** [comment]
It's now at https://espresense.com/Floorplan-Creator/
Sorta works for me, something about the mqtt beacons is causing things to stick.  But my biggest issues is turning on IPS for all the things causes so much CPU on my hass instance it kills it.  I need to move my appdaemon to another box.  That's today's task.

**Comment by stan69b** [comment]
Ah nice, will update the readme today/tonight. What do you mean by "is causing things to stick" ? 
It might be an other subject but i do not manage idle devices, so devices that leaves the home (or shutdown) will remain at the last known position.
